README.rst

```diff
@@ -81,6 +81,10 @@ Here are all options for images:
 		filesystem is created. This option is only used for
 		writeable filesystem types, such as extX, vfat, ubifs and
 		jffs2. This defaults to false.
+:fill:		If this is set to true, then the image file will be filled
+		up to the end of the last partition. This might make the file
+		bigger. This is necessary if the image will be processed by
+		such tools as libvirt, libguestfs or parted.
```

Comment:
I like the description, but this belong in the `hdimage` section below.

Reply:
My bad, will move it down

Reply:
Done